pages/api/v1/votes/batch-votes-cart/getBatchVotesCart.ts

```
@@ -267,6 +267,9 @@ const handler: NextApiHandler<{ votes: IBatchVoteCartResponse[] } | MessageType>
 									payload.status = 'Deciding';
 								}
 							}
+							if (!process.env.AI_SUMMARY_API_KEY) {
+								delete payload?.summary;
```

Biome

[error] Avoid the delete operator which can impact performance.



(lint/performance/noDelete)